data/data/bootstrap/baremetal/systemd/units/master-bmh-update.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Update master BareMetalHosts with introspection data
+Wants=bootkube.service
+After=bootkube.service release-image.service
+
+[Service]
+ExecStart=/usr/local/bin/master-bmh-update.sh
```

Comment:
Since this script exits, does it need to be a OneShot service? I forget the rules.

Reply:
Yeah you're right.